appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js

```diff
@@ -60,11 +67,83 @@ export const postStartDate = async (request, response) => {
 };
 
 /** @type {import('@pins/express').RequestHandler<Response>}  */
-export const getConfirmation = async (request, response) => {
+export const getAddConfirmation = async (request, response) => {
 	const { appealId, appealReference } = request.currentAppeal;
 	const pageContent = startCaseConfirmationPage(appealId, appealReference);
 
 	response.status(200).render('appeals/confirmation.njk', {
 		pageContent
 	});
 };
+
+/** @type {import('@pins/express').RequestHandler<Response>}  */
+export const getChangeDate = async (request, response) => {
+	renderChangeDatePage(request, response);
+};
+
+/**
+ *
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderChangeDatePage = async (request, response) => {
+	const { appealId, appealReference, startedAt, documentationSummary } = request.currentAppeal;
+
+	if (!startedAt || documentationSummary?.lpaQuestionnaire?.status !== 'not_received') {
+		return response.render('app/500.njk');
+	}
+
+	const now = new Date();
+	const today = dateToDisplayDate(now);
+
+	const mappedPageContent = changeDatePage(appealId, appealReference, today);
+
+	return response.render('patterns/display-page.pattern.njk', {
+		pageContent: mappedPageContent
+	});
+};
+
+/** @type {import('@pins/express').RequestHandler<Response>} */
+export const postChangeDate = async (request, response) => {
+	try {
+		const { appealId, documentationSummary } = request.currentAppeal;
+
+		if (documentationSummary?.lpaQuestionnaire?.status !== 'not_received') {
+			return response.render('app/500.njk');
+		}
+
+		const today = new Date();
+		const todayDayMonthYear = {
```

Comment:
You can use the `dateToDayMonthYear` function in `lib/dates.js` to do this

Reply:
Hmm, i don't see one in there. Can you please check again?

Reply:
i see it now after update. done